### PR TITLE
Enhancement: Update packaging meta-data.

### PR DESCRIPTION
**Pull request by dgou**
First stab at updating a bunch of `pyproject.toml` file stuff.
I took a guess/stab at the keywords and classifiers based on a few other PyPI packages found when searching PyPI for "config".

I did a poetry build and then looked at the untard .gz file contents and they looked to have been translated correctly. Open to any all updates/corrections/etc.

### CODE REVIEW

**Review by bradsbrown — APPROVED** [review]
👍

**Review by brolewis — APPROVED** [review]
👍

File: pyproject.toml [hunk 1]
```diff
@@ -2,7 +2,19 @@
 name = "layered-config"
 version = "2.0.0"
 description = "A tool for managing layered config files!"
-authors = ["Ryan Casperson <casperson.ryan@gmail.com>"]
+authors = ["Doug Philips <dgou@mac.com>", "Ryan Casperson <casperson.ryan@gmail.com>"]
```
Comment (rbcasperson) [inline pyproject.toml]: Sorry I had looked at the other PR first. As mentioned there, I think just having you here is most accurate and good with me.

**Review by rbcasperson — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (dgou) [reply pyproject.toml]: WF.

**Comment by dgou** [comment]
Back to @rbcasperson per changes requested.

**Review by rbcasperson — APPROVED** [review]

**Pull request merged**